src/firmware_new/boards/frankie_v1/io/proto_multicast_communication.h

```diff
@@ -0,0 +1,142 @@
+#pragma once
+
+#include "pb.h"
```

Comment:
Is there a full-path for this? If it's a system-include, can we use `<>`?

Reply:
changed to <>